src/context/AuthContext.jsx

```diff
@@ -1,16 +1,43 @@
-import { createContext, useContext, useState } from "react";
+import {createContext, useContext, useReducer, useState} from "react";
 
 const AuthContext = createContext();
 
+function userCredentialHandler(state, action) {
+    console.log(state, action);
+    if (action.email || action.firstName || action.lastName) {
+        localStorage.setItem("FeetzUserDetails", JSON.stringify({
```

Comment:
Instead of using hardcoded strings, may be put them in a object and put the object in a constant.js file and import them here. There are high chances of errors when using hardcoded strings, if you miss a letter, your code breaks.

Reply:
done